company-profiles/kiprosh.md

```diff
@@ -2,47 +2,43 @@
 
-Our happy clients are our best credentials. Our clients are consultancy firms, non-profit organisation, small to medium businesses and startups like [Realvolve](http://realvolve.com/), [CrossRoads](http://crossroads.org.hk/), [GoGoVan](http://gogovan.com.hk/en/), [Thought-Sauce](http://thought-sauce.com/), [Rayku](http://rayku.com/home) and [Renatus](http://www.myrenatus.com/). Here is our [Client List](http://kiprosh.com/clients) with details of our work.
+Our happy clients are our best credentials. Our international clients are consultancy firms, non-profits, medium to large size businesses like Crossroads, Renatus, All American Entertainment, EParamus, Realvolve, GoGoVan, Thought-Sauce, 3Qi-Labs, Rayku and many more. Here is our client list with details of our work.
 
-We also build our own products like the recently launched mobile app [Criticgiri](https://play.google.com/store/apps/details?id=com.kiprosh.criticgiri).
-
-We also organize [Meetups](http://www.meetup.com/Techgravy-Kiprosh) and [Techathons](http://on.fb.me/1NwaeNQ) at our office in Mumbai, India. Check our pics on our [Facebook](https://www.facebook.com/kiprosh/) page. Not bragging, but we have 1000+ followers on [Twitter](https://twitter.com/KiproshCareers) now.
+We also organize [Meetups](http://www.meetup.com/Techgravy-Kiprosh) and [Techathons](http://on.fb.me/1NwaeNQ) at our office in Mumbai, India. Check our pics on our [Facebook](https://www.facebook.com/kiprosh/) page. 
 
 ## Company size
 
-A team of highly technical and collaborative 20+ Craftsmen.
+We are a team of 40+ Craftsmen worldwide.
 
 ## Remote status
 
-We are headquartered in Mumbai, INDIA with an office in Chicago, USA.
-
-In India, our team is spread across multiple cities with co-workers operating from Mumbai and Nagpur. To communicate and collaborate we use [Flowdock](http://flowdock.com/), [Slack](https://slack.com/), [Github](https://github.com/), [Google Hangouts](https://hangouts.google.com/), [Skype](http://www.skype.com/en/) and [GoToMeeting](http://www.gotomeeting.com/). We craft web & mobile applications using [Agile Methodologies](https://en.wikipedia.org/wiki/Agile_software_development).
+We are headquartered in Mumbai, India with an office in Chicago, USA.
 
-We have frequent weekend outings, lunch & dinner parties and lot of fun events.
+In India, our team is spread across multiple cities with team members working from Mumbai, Bangalore, Pune and Nagpur. To communicate and collaborate effectively, we use several tools like [Slack](https://slack.com/), [Github](https://github.com/), [Trello](https://trello.com/) and others. We are remote friendly and calm company.
 
 ## Region
 
 Kiprosh is an international company with office locations in Chicago, USA and Mumbai, INDIA. We have clients from USA, Canada and Hong Kong and thus we work in timezone that has overlap with our client's timezone.
```

Comment:
This part is definitely optional, but I think we can start to use the Region field more effectively in the future.

See:  https://github.com/remoteintech/remote-jobs/issues/485#issuecomment-481939652 (and the rest of the discussion in that issue).

If you want, you can add a simple list to the beginning of the "Region" section, before this existing text:

```markdown
- USA
- India
```

In the future we'll probably implement a filter on the region field, and this will ensure that Kiprush's profile is ready to work with this change.

Reply:
That makes sense @nylen and it's a good suggestion.